pkg/kubelet/server/remotecommand/httpstream.go

```diff
@@ -275,3 +341,19 @@ WaitForStreams:
 
 	return ctx, nil
 }
+
+func handleResizeEvents(stream io.Reader, channel chan<- term.Size) {
+	defer runtime.HandleCrash()
+	//defer close(channel)
+
+	mh := codec.MsgpackHandle{}
```

Comment:
Yeah I'm not going to stick with ugorji. I'll need something that works with the websocket impl too.